### PR TITLE
Styles and functions update for patient application

### PR DESCRIPTION
**Pull request by Shreets**
-> back and skip button to go back to previous page or skip to next page
-> design updates as per figma : https://www.figma.com/file/7XQgaWxsHj7oNPwfWKeLgF/COVIN_India_Project?node-id=902%3A354
-> Only display pages that are yet to be filled

### CODE REVIEW

File: src/pages/CovidHistory.js [hunk 1]
```diff
@@ -73,17 +79,108 @@ const CovidHistory = ({
...
+    setDiagnosed(true);
+  };
+
+  const notDiagnosed = () => {
+    setDiagnosed(false);
+  };
+
+  const handleRadioButton = (value) => {
+    setChecked(value);
+    if (value === 1) {
```
Comment (nastikue) [inline src/pages/CovidHistory.js:100]: what does 1 means here? please move 1 to constants to make code more readable

File: src/pages/CovidHistory.js [hunk 2]
```diff
@@ -73,17 +79,108 @@ const CovidHistory = ({
...
+        </div>
+        <div className="covid-buttons">
+          <span>
+            <input
+              className="covid-button"
+              type="radio"
+              name="yes"
+              value="yes"
+              onClick={yesDiagnosed}
+              checked={state === 1}
```
Comment (nastikue) [inline src/pages/CovidHistory.js:149]: what does 1 means here? please move 1 to constants to make code more readable

File: src/pages/CovidHistory.js [hunk 3]
```diff
@@ -73,17 +79,108 @@ const CovidHistory = ({
...
+            <label for="yes">Yes</label>
+          </span>
+          <span>
+            <input
+              className="covid-button"
+              type="radio"
+              name="no"
+              value="no"
+              onClick={notDiagnosed}
+              checked={state === 2}
```
Comment (nastikue) [inline src/pages/CovidHistory.js:161]: what does 2 means here? please move 2 to constants to make code more readable

File: src/pages/CovidHistory.js [hunk 4]
```diff
@@ -73,17 +79,108 @@ const CovidHistory = ({
...
+
+  const notDiagnosed = () => {
+    setDiagnosed(false);
+  };
+
+  const handleRadioButton = (value) => {
+    setChecked(value);
+    if (value === 1) {
+      setIntakeState({ ...intakeState, covidPositiveEverBefore: true });
+    } else if (value === 2) {
```
Comment (nastikue) [inline src/pages/CovidHistory.js:102]: what does 2 means here? please move 2 to constants to make code more readable

File: src/pages/PatientVitals.js [hunk 5]
```diff
@@ -125,14 +134,49 @@ const PatientVitals = ({
...
+          )}
+        </div>
         <div className="header-wrapper page-hero">
           <div className="main-text">{name}</div>
           <div className="dull-text">{phone}</div>
         </div>
       </div>
       <div className="page-hero dull-text text-center">
-        {day}, {month} {date}, {year}
+        {/* {day}, {month} {date}, {year} */}
```
Comment (nastikue) [inline src/pages/PatientVitals.js:179]: remove commented code

File: src/pages/PatientVitals.js [hunk 6]
```diff
@@ -125,14 +134,49 @@ const PatientVitals = ({
   return (
     <div className="wrapper">
       <div className="second-header">
-        <img src={patient_profile} alt="user-profile" />
+        {(progressedPage === NEW_PATIENT_PAGES.patientInfo ||
```
Comment (nastikue) [inline src/pages/PatientVitals.js:137]: please move condition in a constant, i.e.
`const isShowProgressBar = useMemo(() => [NEW_PATIENT_PAGES.patientInfo, NEW_PATIENT_PAGES.covidHistory, NEW_PATIENT_PAGES.preExistingCondition, NEW_PATIENT_PAGES.allergy, NEW_PATIENT_PAGES.medication].includes(progressedPage), [progressedPage])`

File: src/pages/PatientVitals.js [hunk 7]
```diff
@@ -125,14 +134,49 @@ const PatientVitals = ({
...
+            <div class="progress"></div>
+            <style>{`
+            .progress::after{
+              width : ${(progressedPage - 1) * 20}%;
+            }
+          `}</style>
+          </div>
+        )}
+        <div className="navigation-bar">
+          {(progressedPage === NEW_PATIENT_PAGES.covidHistory ||
```
Comment (nastikue) [inline src/pages/PatientVitals.js:152]: please move condition in a constant

File: src/pages/PatientVitals.js [hunk 8]
```diff
@@ -125,14 +134,49 @@ const PatientVitals = ({
...
+            progressedPage === NEW_PATIENT_PAGES.preExistingCondition ||
+            progressedPage === NEW_PATIENT_PAGES.allergy ||
+            progressedPage === NEW_PATIENT_PAGES.medication) && (
+            <div className="back-button" onClick={goBack}>
+              <img className="nav-img-back" src={back} alt="go back"></img>
+              <span>Back</span>
+            </div>
+          )}
+          {(progressedPage === NEW_PATIENT_PAGES.covidHistory ||
```
Comment (nastikue) [inline src/pages/PatientVitals.js:161]: please move condition in a constant

File: src/pages/PatientPersonalInfo.js [hunk 9]
```diff
@@ -281,26 +283,38 @@ const PatientPersonalInfo = ({
...
+                <div className="weight-wrapper">
+                  <input
+                    type={
+                      info.field === PERSONAL_INFO.weight ? "number" : "text"
+                    }
+                    className="bp"
+                    id={indx}
+                    name={info.field}
+                    value={
+                      info.field === PERSONAL_INFO.firstName
```
Comment (nastikue) [inline src/pages/PatientPersonalInfo.js:295]: move functionality to a callback getValue

Comment (Shreets) [reply src/pages/PatientPersonalInfo.js]: resolved

File: src/pages/PatientVitals.js (hunk 5 above)
Comment (Shreets) [reply src/pages/PatientVitals.js]: resolved

File: src/pages/CovidHistory.js [hunk 10]
```diff
@@ -73,17 +79,108 @@ const CovidHistory = ({
...
+      return moment(intakeState.dateOfDose2Vaccination).format(
+        DATE_FORMAT.yyyymmdd
+      );
+    } else if (field === "dateCovidBefore") {
+      return moment(intakeState.dateCovidBefore).format(DATE_FORMAT.yyyymmdd);
+    }
+  };
+
+  const checkDisabled = (field) => {
+    if (field === "dateOfDose1Vaccination") {
```
[the remark this answers is not in the document]
Comment (Shreets) [reply src/pages/CovidHistory.js]: resolved

File: src/pages/CovidHistory.js [hunk 11]
```diff
@@ -73,17 +79,108 @@ const CovidHistory = ({
...
+      setIntakeState({ ...intakeState, covidPositiveEverBefore: false });
+    }
+  };
+
+  const onBackButtonClick = () => {
+    setProgressedPage(NEW_PATIENT_PAGES.patientInfo);
+  };
+
+  const getValue = (field) => {
+    if (field === "dateOfDose1Vaccination") {
```
[the remark this answers is not in the document]
Comment (Shreets) [reply src/pages/CovidHistory.js]: resolved

File: src/config/index.js [hunk 12]
```diff
@@ -1,5 +1,6 @@
 const config = {
-  apiURL: process.env.REACT_APP_API_URL,
+ apiURL: process.env.REACT_APP_API_URL,
+
```
[the remark this answers is not in the document]
Comment (Shreets) [reply src/config/index.js]: resolved

File: src/App.css [hunk 13]
```diff
@@ -193,32 +348,31 @@ input.bp:last-of-type{
...
 }
-.history-list-content>label{
+.history-list-content > label {
   font-size: 1rem;
   margin-left: 0.4rem;
 }
-.wrapper{
-  min-height: 100vh;
+.wrapper {
+  /* min-height: 100vh; */
```
[the remark this answers is not in the document]
Comment (Shreets) [reply src/App.css]: resolved

File: src/App.css [hunk 14]
```diff
@@ -1,8 +1,28 @@
...
+
+#root .App > div {
+  width: 100%;
+  height: 100%;
+}
+
+#root .App > div > div {
+  width: 100%;
+  height: 100%;
+}
```
[the remark this answers is not in the document]
Comment (Shreets) [reply src/App.css]: resolved

**Review by nastikue — APPROVED** [review]

**Pull request merged**